contracts/RegistrarController.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: UNLICENSED
+
+import "./interfaces/IRegistrarController.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+pragma solidity ^0.8;
+
+/// @title A simple domain registry
+/// @author Sergey Nesterov (Sergio-Prog)
+contract RegistrarController is IRegistrarController, Ownable {
+    uint public domainPrice;
+
+    mapping (string => address) private domainRecords;
+
+    constructor(address _owner, uint _domainPrice) Ownable(_owner) {
+        domainPrice = _domainPrice;
+    }
+
+    /// @inheritdoc IRegistrarController
```

Comment:
// line 19 and other same comments
// As best practice, you should use comment for contract, and inherit it from ownable interface
// NatSpec comments should be used for all functions and state variables
// - 1 point